Entrega 01-10 Ismael Cobo

ismaelcobo opened the pull request.

Commit:
CreaciónCarpetas

ismaelcobo committed:
Create coboIsmael

ismaelcobo committed:
Delete entregas/coboIsmael

ismaelcobo committed:
Delete entregas/CoboIsmael directory

ismaelcobo committed:
Create coboIsmael

ismaelcobo committed:
Delete entregas/coboIsmael

ismaelcobo committed:
Create coboIsmael

ismaelcobo committed:
Delete entregas/coboIsmael

ismaelcobo committed:
Create a

ismaelcobo committed:
Delete entregas/coboIsmael directory

ismaelcobo committed:
micarpetadeentregas

ismaelcobo committed:
micarpetaorigen

ismaelcobo committed:
micarpetadestino

ismaelcobo committed:
misarchivosmodificados

ismaelcobo committed:
misarchivosmodificados

ismaelcobo committed:
Delete entregas/coboIsmael directory

ismaelcobo committed:
micarpetadeentregas

ismaelcobo committed:
micarpetaorigen

ismaelcobo committed:
micarpetadestino

ismaelcobo committed:
textosmodificados

ismaelcobo committed:
archivosmovidos

ismaelcobo committed:
archivosmovidos

ismaelcobo committed:
micarpetaorigen

ismaelcobo committed:
textosmodificadosx2

ismaelcobo committed:
Create src

ismaelcobo committed:
Delete entregas/coboIsmael/src

ismaelcobo committed:
Create a

ismaelcobo committed:
Delete entregas/coboIsmael/scr directory

Commit:
Carpeta src creada

Commit:
Carpeta src creada

Commit:
Carpeta src creada

Commit:
Carpeta src creada

ismaelcobo committed:
Create ArchivoCreación.txt

ismaelcobo committed:
Delete entregas/coboIsmael/HolaMundo.java directory

ismaelcobo committed:
Delete entregas/coboIsmael/src/HolaMundo

ismaelcobo committed:
Create HolaMundo.java

Commit:
pull request tarea

mmasias committed:
Update aDiaDeHoy.md

mmasias commented on entregas/coboIsmael/src/HolaMundo.java:
Los nombres de clases empiezan con mayúscula

mmasias committed:
Update aDiaDeHoy.md

ismaelcobo committed:
Pull NumerosAleatorios

ismaelcobo committed:
Push NumerosAleatorios

mmasias committed:
Actualización de retos y detalles de entrega

- Eliminado el texto genérico sobre la entrega de retos y sus fechas límite.
- Actualizada la tabla en el README de retos con información específica de entrega para el Reto 001 y el nuevo Reto 002, incluyendo fechas de entrega y ramas específicas.
- Añadido el archivo reto002.md con la estructura inicial y la fecha de entrega especificada.

mmasias committed:
fix: Titulo

mmasias committed:
Dos ejemplos...

mmasias committed:
Tres ejemplos ternarios

mmasias committed:
Enunciado de los ejemplos

mmasias committed:
Enlace en la página de ejercicios

mmasias committed:
Incluye salida esperada en contenido de ejercicios

mmasias committed:
Update ClasificacionEdad.md

mmasias committed:
Definición de reto 001

mmasias committed:
Refinamos requisito (Human vs Computer)

mmasias committed:
Una imagen...

ismaelcobo committed:
pusheo JuegoPiedraPapelTijera

ismaelcobo committed:
pusheo JuegoPiedraPapelTijera

Commit:
pusheo JuesgosFor

Commit:
pusheo ElCaracol

ismaelcobo committed:
Delete entregas/coboIsmael/src/ElCaracol.java

ismaelcobo committed:
pusheo Caracol

ismaelcobo committed:
pusheo Caracol

ismaelcobo committed:
pusheo CaracolFinal

mmasias committed:
Código!

mmasias committed:
Caracol ternario!

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Recopilación y ordenado de ejercicios

mmasias committed:
Create .gitignore

mmasias committed:
Ejercicios++

mmasias committed:
Update README.md

mmasias committed:
🪨📄✂️🦎🖖🏻

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Un dia en la vida de Marco
Pendiente:
- La madre
- Los dias

Commit:
pusheo ViajeMarco

mmasias committed:
Solucion hecha en clase

mmasias committed:
Update ViajeMarco.java

Es más lógico expresarlo así:

seHanEncontrado si la distancia entre Marco y su madre es menor o igual a cero.

mmasias committed:
Propuesta de solución fullTernario 💪🏼💪🏼💪🏼

mmasias committed:
Actualización con soluciones

mmasias committed:
CarreraCamellos hecho en clase

mmasias committed:
Publicada la solución propuesta de CarreraCamellos

mmasias committed:
Extensión

Commit:
Pusheo CarreraCamellos

mmasias committed:
Avance de clase. Inicio de solución de la Batalla

mmasias committed:
Update README.md

mmasias committed:
Versión minimalista
Vídeo de construcción en https://www.youtube.com/watch?v=bebuAZLYAVE

mmasias committed:
Update README.md

mmasias committed:
Ejercicio del centro comercial
Además incluyo una corrección en el ejercicio de la carrera de camellos.

mmasias committed:
Solución (hecha en clase) del centro comercial

mmasias committed:
Fix & version ASCII

mmasias committed:
Update README.md

mmasias committed:
Fix, inclusión de hora/minuto y animación

mmasias committed:
Segundo intento

mmasias committed:
3rd

mmasias committed:
Enlaces

mmasias committed:
Centro comercial resuelto con métodos estáticos
No he puesto commits individuales :( Disculparme

ismaelcobo committed:
Pusheo AdivinaElNumero

ismaelcobo committed:
Pusheo AdivinaElNumero

mmasias committed:
eP2

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Ejercicio de temperatura de 7 dias
- Con métodos estáticos
- Adecuado reparto de responsabilidades

mmasias committed:
Refactoriza manejo de temperaturas a matrices

- Cambia variables individuales de temperatura por una matriz.
- Actualiza métodos calcularMinima, calcularMaxima y calcularMedia para usar la nueva matriz.
- Simplifica el código de solicitud de temperaturas con un bucle for.

mmasias committed:
Pequeñas anécdotas acerca de una matriz

mmasias committed:
Cambio extra

ismaelcobo committed:
Pusheo Matrices

JavierLianoRioz committed:
Update README.md

mmasias committed:
Merge pull request #689 from JavierLianoRioz/patch-1

Modificación del Readme

mmasias committed:
pyMundo y su solución

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

mmasias committed:
fixLink

ismaelcobo committed:
Merge branch 'mmasias:main' into main
